WordPress/Classes/Services/PostRepository+Helpers.swift

```diff
@@ -4,8 +4,10 @@ import WordPressKit
 extension RemotePostCreateParameters {
     /// Initializes the parameters required to create the given post.
     init(post: AbstractPost) {
-        self.init(status: (post.status ?? .draft).rawValue)
-
+        self.init(
```

Comment:
Just letting you know, it would great to get this merged to the trunk soon since we get a compiler and unit test errors when integrating the latest commit of `WPKit-iOS` to the project. 

It's not the first time we've had a situation such as this 🤔 maybe it would make sense not to merge `WPKit-iOS` changes until `WordPress-iOS` repo PR is prepared to be merged to `trunk`. Although it's far from ideal.